### PR TITLE
fix typos in README

### PR DESCRIPTION
**Pull request by andy-neuma**
SUMMARY:
* update README to fix typos

TEST PLAN: 
team review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,21 +33,16 @@ pip install nm-vllm[sparse] --extra-index-url https://pypi.neuralmagic.com/simpl
 
 ### Docker
 
-The `nm-vllm-ent` [container registry](https://github.com/neuralmagic/nm-vllm-certs/pkgs/container/nm-vllm-ent) includes premade docker images.
+The `nm-vllm-certs` [container registry](https://github.com/neuralmagic/nm-vllm-certs/pkgs/container/nm-vllm-certs) includes premade docker images.
```
Comment (dhuangnm) [inline README.md]: We use nm-vllm-ent as the docker image name so the old URL is actually correct, please change this back. The docker image is published in the nm-vllm-certs repo.

File: README.md [hunk 2]
```diff
@@ -28,21 +33,16 @@ pip install nm-vllm[sparse] --extra-index-url https://pypi.neuralmagic.com/simpl
...
 
 Launch the OpenAI-compatible server with:
 
 ```bash
 MODEL_ID=Qwen/Qwen2-0.5B-Instruct
-docker run --gpus all --shm-size 2g ghcr.io/neuralmagic/nm-vllm-ent:latest --model $MODEL_ID
+docker run --gpus all --shm-size 2g ghcr.io/neuralmagic/nm-vllm-certs:latest --model $MODEL_ID
```
Comment (dhuangnm) [inline README.md]: This is incorrect, please use nm-vllm-ent as the docker image name.

**Review by dhuangnm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andy-neuma) [reply README.md]: good catch! i'll revert.

File: README.md (hunk 2 above)
Comment (andy-neuma) [reply README.md]: yep, good catch.

File: README.md [hunk 3]
```diff
@@ -2,13 +2,18 @@
 
 
 ## Overview
-The `nm-vllm` packages published in this repository are Neuralmagic Enterprise Edition of [vLLM](https://github.com/vllm-project/vllm). Packages are versioned Python wheels and docker images. These are released as "production level" official releases and "beta level" nightly releases.
+The `nm-vllm` packages published in this repository are Neural Magic Enterprise Editions of [vLLM](https://github.com/vllm-project/vllm). Packages are versioned Python wheels and docker images. These are released as "production level" official releases and "beta level" nightly releases.
 
-Official releases are made at the discretion of Neuralmagic, but typically track with `vllm` releases. These wheels are available via the official PyPI as well as [Neuralmagic's PyPI](https://pypi.neuralmagic.com).
+Official releases are made at the discretion of Neural Magic, but typically track with `vllm` releases. These wheels are available via the official PyPI as well as [Neuralmagic's PyPI](https://pypi.neuralmagic.com).
```
Comment (dhuangnm) [inline README.md]: Should the "Neuralmagic's PyPI" be changed to "Neural Magic's PyPI" based on the change in this line?

**Review by dhuangnm — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (andy-neuma) [reply README.md]: yeah, i think so

**Review by dhuangnm — DISMISSED** [review]
LGTM, thanks.

File: README.md [hunk 4]
```diff
@@ -2,11 +2,16 @@
 
 
 ## Overview
-The `nm-vllm` packages published in this repository are Neuralmagic Enterprise Edition of [vLLM](https://github.com/vllm-project/vllm). Packages are versioned Python wheels and docker images. These are released as "production level" official releases and "beta level" nightly releases.
+The `nm-vllm` packages published in this repository are Neural Magic Enterprise Editions of [vLLM](https://github.com/vllm-project/vllm). Packages are versioned Python wheels and docker images. These are released as "production level" official releases and "beta level" nightly releases.
```
Comment (jeanniefinks) [inline README.md]:
```suggestion
The `nm-vllm` packages published in this repository are Neural Magic Enterprise Editions of [vLLM](https://github.com/vllm-project/vllm). Packages are versioned Python wheels and Docker images. These are released as "production level" official releases and "beta level" nightly releases.
```

**Review by jeanniefinks — DISMISSED** [review]
added a minor nit that if you could fix, no need to re-review.

**Pull request merged**